packages/awslogs/spec

```diff
@@ -2,15 +2,4 @@
 name: awslogs
 dependencies:
 files:
-- awscli-cwlogs-1.3.3.tar.gz
-- aws-cli/awscli-bundle.zip
-- Python-2.7.10.tgz
-- setuptools-21.0.0.tar.gz
-- pip-8.1.1.tar.gz
-- six-1.10.0-py2.py3-none-any.whl
-- py-dateutil-2.2.tar.gz
-- bzip2-1.0.6.tar.gz
-- openssl-1.0.2h.tar.gz
-- zlib-1.2.8.tar.gz
-- sqlite-autoconf-3130000.tar.gz
-- pysqlite-2.8.2.tar.gz
+- .gitkeep
```

Comment:
This is gross, but is literally the recommend approach in a pivotal blog post: https://blog.pivotal.io/labs/labs/how-to-create-a-bosh-release-of-a-dns-server